### PR TITLE
[IMP] mrp_product_profit: new total view

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

**Comment by oihane** [comment]
Nueva vista:
image

**Comment by oihane** [comment]
@avanzosc/developers review please

**Comment by esthermm** [comment]
:+1:

File: mrp_product_profit/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+#. module: mrp_product_profit
+#: field:mrp.production,routing_commercial:0
+msgid "Commercial (Routing)"
+msgstr "Márgen comercial"
+
+#. module: mrp_product_profit
+#: field:sale.order.line,commercial_percent:0
+msgid "Commercial percentage"
+msgstr "Porcentad de márgen comercial"
```
Comment (agaldona) [inline mrp_product_profit/i18n/es.po]: ??

**Review by agaldona — COMMENTED** [review]
:+1:

**Review by Daniel-CA — APPROVED** [review]
:+1: When translation is changed

**Comment by oihane** [comment]
Depends on #335

**Review by agaldona — APPROVED** [review]
:+1:

**Pull request merged**